ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/js/button.js

```diff
@@ -11,7 +11,8 @@ $(function() {
                 document.querySelector('.logout').addEventListener('click', function(){
                     console.log("on click event listner code ")
```

Comment:
@DeepaliAgrwl : dont need console logs here

Reply:
I have removed the console statement. Please check.